### ISSUE TITLE
`termux-call-log` & `termux-sms-list` add dual sim feature

### ISSUE THREAD

**Issue by wzaw1** [termux/termux-api#441]
**Feature description**
I know there already is option in `termux-sms-send` to choose which sim card do You wanna use, so maybe it's quite easy task. It would be great if `call-log` and `sms-list`  will have option to choose/sort sms/call log depending on sim card. Our scripts could be even more advanced :).

_Btw. Big Thanks for You guys, great job! Termux and termux-api are awesome. Both made my work much more efficient and saved me a lot of time._

**Comment by wzaw1** [comment]
[https://stackoverflow.com/questions/62568563/identify-call-logs-from-sim1-and-sim2](url) maybe that will help?

**Comment by andremarcais** [comment]
Yeah, dual sim support for all of the telephony related tools would be nice. I just haven't gotten around to implementing it yet. I might get to it some time this summer. (Or is someone else already working on this?)
